src/main/java/org/sonarlint/intellij/issue/IssueManager.java

```diff
@@ -252,7 +255,8 @@ public void insertNewIssue(VirtualFile file, LiveIssue liveIssue) {
   }
 
   private void notifyAboutIssueChangeForFile(VirtualFile file) {
-    myProject.getMessageBus().syncPublisher(IssueStoreListener.SONARLINT_ISSUE_STORE_TOPIC).fileChanged(file, liveIssueCache.getLive(file));
+    Collection<LiveIssue> issues = liveIssueCache.getLive(file) == null ? new ArrayList<>() : liveIssueCache.getLive(file);
```

Comment:
This is not correct. `liveIssueCache.getLive(file)` can return non null the first time, and `null` the second time...